client/src/utils/checkTokenIsAuth.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+import { SET_LOGIN } from '../store/actions/adminLoginActions';
+import { UPDATE_SNACKBAR } from '../store/actions/snackbarActions';
+
+const isTokenAuthed = async (dispatch, currentAuth) => {
+  try {
+    const token = window.localStorage.getItem('jwt_token');
+    const res = await axios.get('/admins', {
+      headers: { Authorization: `JWT ${token}` },
+    });
+    if (res && res.status === 200 && currentAuth === false) {
+      const payload = {
+        auth: true,
+      };
+      dispatch({ type: SET_LOGIN, payload });
+      return true;
+    }
+  } catch (err) {
+    const res = err.response;
+
+    if (res && res.status === 401 && currentAuth === true) {
```

Comment:
can you save this as a more declarative variable

Reply:
Agreed. Fixed!